packages/web-react/src/components/Container/useContainerStyleProps.ts

```diff
@@ -10,12 +10,15 @@ export interface ContainerStyles {
 }
 
 export function useContainerStyleProps(props: SpiritContainerProps): ContainerStyles {
-  const { isFluid, ...modifiedProps } = props;
+  const { isFluid, size, ...modifiedProps } = props;
```

Comment:
either we are not correct in the Docs or the default is xlarge. It should be actually set.